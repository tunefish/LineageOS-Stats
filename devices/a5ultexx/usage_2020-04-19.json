{
  "14.1-20170114-unofficial-a5ultexx": 7,
  "14.1-20170216-unofficial-a5ultexx": 80,
  "14.1-20170817-unofficial-a5ultexx": 47,
  "14.1-20170822-unofficial-a5ultexx": 9,
  "14.1-20170906-unofficial-a5ultexx": 98,
  "14.1-20171231_020714-unofficial-a5ultexx": 39,
  "14.1-20180206-unofficial-a5ultexx": 1,
  "14.1-20180615-unofficial-a5ultexx": 1,
  "14.1-20180820-unofficial-a5ultexx": 9,
  "14.1-20180825-unofficial-a5ultexx": 1,
  "14.1-20180826-unofficial-a5ultexx": 1,
  "14.1-20180829-unofficial-a5ultexx": 1,
  "14.1-20190708-unofficial-a5ultexx": 17,
  "14.1-20200118-unofficial-a5ultexx": 1,
  "15.1-20180210-unofficial-a5ultexx": 111,
  "16.0-20181015-unofficial-a5ultexx": 1,
  "16.0-20181102-unofficial-a5ultexx": 7,
  "16.0-20190129-unofficial-a5ultexx": 1,
  "16.0-20190220-unofficial-a5ultexx": 1,
  "16.0-20190225-unofficial-a5ultexx": 14,
  "16.0-20190328-unofficial-a5ultexx": 13,
  "16.0-20190427-unofficial-a5ultexx": 384,
  "17.0-20190921-unofficial-a5ultexx": 2,
  "17.0-20191005-unofficial-a5ultexx": 50,
  "17.0-20191024-unofficial-a5ultexx": 96,
  "17.0-20191118-unofficial-a5ultexx": 4,
  "17.0-20191217-unofficial-a5ultexx": 1,
  "17.1-20200111-unofficial-a5ultexx": 29,
  "17.1-20200305-unofficial-a5ultexx": 2
}