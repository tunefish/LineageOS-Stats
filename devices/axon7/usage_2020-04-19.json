{
  "14.1-20170203-unofficial-axon7": 7,
  "14.1-20170213-nightly-axon7": 1,
  "14.1-20170327-nightly-axon7": 1,
  "14.1-20170501-nightly-axon7": 1,
  "14.1-20170508-nightly-axon7": 1,
  "14.1-20170524-unofficial-axon7": 1,
  "14.1-20170619-nightly-axon7": 1,
  "14.1-20170904-nightly-axon7": 1,
  "14.1-20171002-nightly-axon7": 1,
  "14.1-20171016-nightly-axon7": 3,
  "14.1-20171023-nightly-axon7": 3,
  "14.1-20171113-nightly-axon7": 1,
  "14.1-20171127-nightly-axon7": 1,
  "14.1-20171204-nightly-axon7": 1,
  "14.1-20180101-nightly-axon7": 2,
  "14.1-20180108-nightly-axon7": 1,
  "14.1-20180212-nightly-axon7": 1,
  "14.1-20180220-nightly-axon7": 2,
  "14.1-20180228-nightly-axon7": 3,
  "14.1-20180307-nightly-axon7": 3,
  "14.1-20180314-microg-axon7": 1,
  "14.1-20180314-nightly-axon7": 2,
  "14.1-20180321-nightly-axon7": 1,
  "14.1-20180328-nightly-axon7": 8,
  "14.1-20180410-nightly-axon7": 1,
  "14.1-20180418-nightly-axon7": 1,
  "14.1-20180425-nightly-axon7": 2,
  "14.1-20180503-nightly-axon7": 1,
  "14.1-20180509-nightly-axon7": 5,
  "14.1-20180516-nightly-axon7": 2,
  "14.1-20180523-nightly-axon7": 1,
  "14.1-20180530-nightly-axon7": 2,
  "14.1-20180606-nightly-axon7": 1,
  "14.1-20180613-nightly-axon7": 4,
  "14.1-20180620-nightly-axon7": 4,
  "14.1-20180704-nightly-axon7": 1,
  "14.1-20180718-nightly-axon7": 1,
  "14.1-20180801-nightly-axon7": 6,
  "14.1-20180815-nightly-axon7": 5,
  "14.1-20180822-nightly-axon7": 3,
  "14.1-20180829-nightly-axon7": 6,
  "14.1-20180905-nightly-axon7": 5,
  "14.1-20180912-nightly-axon7": 9,
  "14.1-20180919-nightly-axon7": 44,
  "15.1-20180302-unofficial-axon7": 3,
  "15.1-20180609-unofficial-axon7": 1,
  "15.1-20180615-unofficial-axon7": 1,
  "15.1-20180625-unofficial-axon7": 18,
  "15.1-20180810-unofficial-axon7": 1,
  "15.1-20180911-unofficial-axon7": 4,
  "15.1-20180925-nightly-axon7": 116,
  "15.1-20181001-nightly-axon7": 2,
  "15.1-20181008-nightly-axon7": 7,
  "15.1-20181015-nightly-axon7": 2,
  "15.1-20181019-unofficial-axon7": 1,
  "15.1-20181022-nightly-axon7": 3,
  "15.1-20181029-nightly-axon7": 3,
  "15.1-20181029-unofficial-axon7": 1,
  "15.1-20181105-nightly-axon7": 3,
  "15.1-20181110-unofficial-axon7": 3,
  "15.1-20181117-nightly-axon7": 1,
  "15.1-20181122-nightly-axon7": 1,
  "15.1-20181124-nightly-axon7": 1,
  "15.1-20181125-nightly-axon7": 1,
  "15.1-20181127-nightly-axon7": 1,
  "15.1-20181129-nightly-axon7": 1,
  "15.1-20181203-nightly-axon7": 1,
  "15.1-20181206-nightly-axon7": 3,
  "15.1-20181208-nightly-axon7": 2,
  "15.1-20181213-unofficial-axon7": 1,
  "15.1-20181214-nightly-axon7": 1,
  "15.1-20181215-nightly-axon7": 2,
  "15.1-20181216-nightly-axon7": 2,
  "15.1-20181218-nightly-axon7": 1,
  "15.1-20181219-nightly-axon7": 1,
  "15.1-20181223-nightly-axon7": 4,
  "15.1-20181225-nightly-axon7": 1,
  "15.1-20181229-nightly-axon7": 2,
  "15.1-20181230-nightly-axon7": 2,
  "15.1-20190102-nightly-axon7": 1,
  "15.1-20190106-nightly-axon7": 1,
  "15.1-20190108-nightly-axon7": 1,
  "15.1-20190114-nightly-axon7": 1,
  "15.1-20190115-nightly-axon7": 1,
  "15.1-20190116-nightly-axon7": 3,
  "15.1-20190117-nightly-axon7": 1,
  "15.1-20190119-nightly-axon7": 1,
  "15.1-20190120-nightly-axon7": 2,
  "15.1-20190130-unofficial-axon7": 2,
  "15.1-20190131-nightly-axon7": 1,
  "15.1-20190201-nightly-axon7": 1,
  "15.1-20190202-nightly-axon7": 2,
  "15.1-20190204-nightly-axon7": 2,
  "15.1-20190205-nightly-axon7": 1,
  "15.1-20190214-nightly-axon7": 2,
  "15.1-20190215-nightly-axon7": 2,
  "15.1-20190218-nightly-axon7": 1,
  "15.1-20190219-nightly-axon7": 1,
  "15.1-20190221-nightly-axon7": 1,
  "15.1-20190222-nightly-axon7": 1,
  "15.1-20190223-nightly-axon7": 2,
  "15.1-20190224-nightly-axon7": 6,
  "15.1-20190224-unofficial-axon7": 4,
  "15.1-20190301-nightly-axon7": 6,
  "15.1-20190308-nightly-axon7": 6,
  "15.1-20190315-nightly-axon7": 1,
  "15.1-20190322-nightly-axon7": 7,
  "15.1-20190405-nightly-axon7": 6,
  "15.1-20190410-unofficial-axon7": 1,
  "15.1-20190412-nightly-axon7": 12,
  "15.1-20190412-unofficial-axon7": 1,
  "15.1-20190419-nightly-axon7": 10,
  "15.1-20190426-nightly-axon7": 25,
  "15.1-20190507-microg-axon7": 1,
  "15.1-20190514-microg-axon7": 7,
  "15.1-20190517-nightly-axon7": 172,
  "15.1-20190525-unofficial-axon7": 4,
  "15.1-20190710-unofficial-axon7": 3,
  "15.1-20190713-unofficial-axon7": 2,
  "15.1-20190817-unofficial-axon7": 1,
  "15.1-20190820-unofficial-axon7": 1,
  "15.1-20190826-unofficial-axon7": 2,
  "15.1-20190912-unofficial-axon7": 1,
  "15.1-20190919-unofficial-axon7": 2,
  "15.1-20191015-unofficial-axon7": 3,
  "15.1-20191117-unofficial-axon7": 2,
  "15.1-20191217-unofficial-axon7": 81,
  "15.1-20200116-unofficial-axon7": 1,
  "15.1-20200202-unofficial-axon7": 1,
  "15.1-20200326-unofficial-axon7": 1,
  "16.0-20180909-unofficial-axon7": 3,
  "16.0-20180918-unofficial-axon7": 24,
  "16.0-20181114-unofficial-axon7": 10,
  "16.0-20190312-unofficial-axon7": 32,
  "16.0-20190728-unofficial-axon7": 5,
  "16.0-20190729-unofficial-axon7": 233,
  "16.0-20190922-unofficial-axon7": 3,
  "16.0-20200119-unofficial-axon7": 1,
  "16.0-20200220-unofficial-axon7": 1,
  "17.1-20200202-unofficial-axon7": 1,
  "17.1-20200203-unofficial-axon7": 3,
  "nuclearom-v-official[7.1.2]--20170703-1123": 6
}