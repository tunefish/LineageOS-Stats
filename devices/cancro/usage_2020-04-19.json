{
  "11-20190526-unofficial-cancro": 7,
  "13.0-20170202-unofficial-cancro": 4,
  "13.0-20170217-unofficial-cancro": 2,
  "13.0-20170319-unofficial-cancro": 2,
  "13.0-20170321-unofficial-cancro": 19,
  "13.0-20170428-unofficial-cancro": 1,
  "13.0-20170721-unofficial-cancro": 297,
  "14.1-20161111-nightly-cancro": 1,
  "14.1-20170109-unofficial-cancro": 1,
  "14.1-20170112-unofficial-cancro": 4,
  "14.1-20170114-unofficial-cancro": 2,
  "14.1-20170123-nightly-cancro": 1,
  "14.1-20170126-unofficial-cancro": 2,
  "14.1-20170130-nightly-cancro": 2,
  "14.1-20170206-nightly-cancro": 5,
  "14.1-20170213-nightly-cancro": 7,
  "14.1-20170214-unofficial-cancro": 168,
  "14.1-20170221-unofficial-cancro": 21,
  "14.1-20170227-nightly-cancro": 2,
  "14.1-20170306-nightly-cancro": 2,
  "14.1-20170313-nightly-cancro": 2,
  "14.1-20170320-nightly-cancro": 3,
  "14.1-20170327-nightly-cancro": 1,
  "14.1-20170403-nightly-cancro": 3,
  "14.1-20170410-nightly-cancro": 4,
  "14.1-20170424-nightly-cancro": 1,
  "14.1-20170501-nightly-cancro": 4,
  "14.1-20170508-nightly-cancro": 5,
  "14.1-20170515-nightly-cancro": 2,
  "14.1-20170521-unofficial-cancro": 1,
  "14.1-20170522-nightly-cancro": 5,
  "14.1-20170529-nightly-cancro": 2,
  "14.1-20170605-nightly-cancro": 1,
  "14.1-20170612-nightly-cancro": 4,
  "14.1-20170619-nightly-cancro": 1,
  "14.1-20170625-unofficial-cancro": 72,
  "14.1-20170626-nightly-cancro": 3,
  "14.1-20170703-nightly-cancro": 3,
  "14.1-20170710-nightly-cancro": 3,
  "14.1-20170717-nightly-cancro": 6,
  "14.1-20170724-nightly-cancro": 4,
  "14.1-20170731-nightly-cancro": 1,
  "14.1-20170807-nightly-cancro": 1,
  "14.1-20170814-nightly-cancro": 3,
  "14.1-20170821-nightly-cancro": 1,
  "14.1-20170826-unofficial-cancro": 1,
  "14.1-20170828-nightly-cancro": 1,
  "14.1-20170830-unofficial-cancro": 1,
  "14.1-20170904-nightly-cancro": 4,
  "14.1-20170911-nightly-cancro": 7,
  "14.1-20170911-unofficial-cancro": 459,
  "14.1-20170918-nightly-cancro": 6,
  "14.1-20170925-nightly-cancro": 3,
  "14.1-20171009-nightly-cancro": 10,
  "14.1-20171016-nightly-cancro": 1,
  "14.1-20171023-nightly-cancro": 1,
  "14.1-20171030-nightly-cancro": 5,
  "14.1-20171106-nightly-cancro": 5,
  "14.1-20171113-nightly-cancro": 5,
  "14.1-20171120-nightly-cancro": 7,
  "14.1-20171204-nightly-cancro": 4,
  "14.1-20171211-nightly-cancro": 250,
  "14.1-20171218-nightly-cancro": 2,
  "14.1-20171225-nightly-cancro": 5,
  "14.1-20180101-nightly-cancro": 6,
  "14.1-20180108-nightly-cancro": 14,
  "14.1-20180122-nightly-cancro": 3,
  "14.1-20180129-nightly-cancro": 3,
  "14.1-20180205-nightly-cancro": 2,
  "14.1-20180212-nightly-cancro": 7,
  "14.1-20180220-nightly-cancro": 6,
  "14.1-20180228-nightly-cancro": 7,
  "14.1-20180307-nightly-cancro": 3,
  "14.1-20180314-nightly-cancro": 3,
  "14.1-20180321-nightly-cancro": 2,
  "14.1-20180328-nightly-cancro": 2,
  "14.1-20180402-unofficial-cancro": 573,
  "14.1-20180404-nightly-cancro": 9,
  "14.1-20180410-nightly-cancro": 10,
  "14.1-20180418-nightly-cancro": 2,
  "14.1-20180425-nightly-cancro": 4,
  "14.1-20180503-nightly-cancro": 3,
  "14.1-20180509-nightly-cancro": 5,
  "14.1-20180516-nightly-cancro": 72,
  "14.1-20180523-nightly-cancro": 7,
  "14.1-20180530-nightly-cancro": 8,
  "14.1-20180606-nightly-cancro": 1,
  "14.1-20180613-nightly-cancro": 5,
  "14.1-20180620-nightly-cancro": 1,
  "14.1-20180627-nightly-cancro": 38,
  "14.1-20180711-nightly-cancro": 3,
  "14.1-20180718-nightly-cancro": 3,
  "14.1-20180725-nightly-cancro": 3,
  "14.1-20180725-unofficial-cancro": 104,
  "14.1-20180801-nightly-cancro": 10,
  "14.1-20180808-nightly-cancro": 10,
  "14.1-20180815-nightly-cancro": 3,
  "14.1-20180822-nightly-cancro": 6,
  "14.1-20180827_115637-unofficial-cancro": 1,
  "14.1-20180829-nightly-cancro": 11,
  "14.1-20180905-nightly-cancro": 10,
  "14.1-20180912-nightly-cancro": 7,
  "14.1-20180919-nightly-cancro": 7,
  "14.1-20180926-nightly-cancro": 11,
  "14.1-20181003-nightly-cancro": 17,
  "14.1-20181010-nightly-cancro": 63,
  "14.1-20181017-nightly-cancro": 5,
  "14.1-20181024-nightly-cancro": 13,
  "14.1-20181031-nightly-cancro": 11,
  "14.1-20181101-unofficial-cancro": 2018,
  "14.1-20181107-nightly-cancro": 1658,
  "14.1-20181201-nightly-cancro": 307,
  "14.1-20181206-unofficial-cancro": 1,
  "14.1-20181228-unofficial-cancro": 2,
  "14.1-20190101-nightly-cancro": 77,
  "14.1-20190122-unofficial-cancro": 1,
  "14.1-20190201-nightly-cancro": 342,
  "14.1-20190214-microg-cancro": 5,
  "14.1-20190302-microg-cancro": 14,
  "14.1-20190305_022204-unofficial-cancro": 20,
  "14.1-20190306_092442-unofficial-cancro": 1,
  "14.1-20190422-unofficial-cancro": 101,
  "14.1-20190425-unofficial-cancro": 5,
  "14.1-20190514-unofficial-cancro": 15,
  "14.1-20190515-unofficial-cancro": 3,
  "14.1-20190516-unofficial-cancro": 2,
  "14.1-20190519-unofficial-cancro": 1,
  "14.1-20190520-unofficial-cancro": 15,
  "14.1-20190531-unofficial-cancro": 7,
  "14.1-20190609-unofficial-cancro": 8,
  "14.1-20190614-unofficial-cancro": 1,
  "14.1-20190702-unofficial-cancro": 4,
  "14.1-20190705-unofficial-cancro": 1,
  "14.1-20190720-unofficial-cancro": 44,
  "14.1-20190722-unofficial-cancro": 157,
  "14.1-20190802-unofficial-cancro": 6,
  "14.1-20190811-unofficial-cancro": 1,
  "14.1-20190812-unofficial-cancro": 12,
  "14.1-20190814-unofficial-cancro": 1,
  "14.1-20190816-unofficial-cancro": 6,
  "14.1-20190825-unofficial-cancro": 251,
  "14.1-20190910-unofficial-cancro": 8,
  "14.1-20190912-unofficial-cancro": 1,
  "14.1-20190916-unofficial-cancro": 7,
  "14.1-20190918-unofficial-cancro": 136,
  "14.1-20190924-unofficial-cancro": 1,
  "14.1-20190925-unofficial-cancro": 6,
  "14.1-20190929-unofficial-cancro": 189,
  "14.1-20191014-unofficial-cancro": 6,
  "14.1-20191030-unofficial-cancro": 1,
  "14.1-20191031-unofficial-cancro": 274,
  "14.1-20191105-unofficial-cancro": 1,
  "14.1-20191113-unofficial-cancro": 270,
  "14.1-20191126-unofficial-cancro": 1,
  "14.1-20191127_033131-unofficial-cancro": 36,
  "14.1-20191127_074450-unofficial-cancro": 75,
  "14.1-20191128-unofficial-cancro": 24,
  "14.1-20191129-unofficial-cancro": 97,
  "14.1-20191202-unofficial-cancro": 8,
  "14.1-20191203-unofficial-cancro": 5,
  "14.1-20191204-unofficial-cancro": 3,
  "14.1-20191206-unofficial-cancro": 1,
  "14.1-20191209-unofficial-cancro": 25,
  "14.1-20191209-unofficial-libra": 2,
  "14.1-20191213-unofficial-cancro": 18,
  "14.1-20191217-unofficial-cancro": 268,
  "14.1-20191219-unofficial-cancro": 1,
  "14.1-20191223-unofficial-cancro": 1,
  "14.1-20191226-unofficial-cancro": 10,
  "14.1-20191229-unofficial-libra": 2,
  "14.1-20191230-unofficial-cancro": 8,
  "14.1-20200102-unofficial-cancro": 3,
  "14.1-20200103-unofficial-cancro": 473,
  "14.1-20200106-unofficial-cancro": 4,
  "14.1-20200107_041704-unofficial-cancro": 29,
  "14.1-20200108-unofficial-cancro": 1,
  "14.1-20200109-unofficial-cancro": 5,
  "14.1-20200110-unofficial-cancro": 9,
  "14.1-20200111-unofficial-cancro": 1,
  "14.1-20200112-unofficial-cancro": 2,
  "14.1-20200113-unofficial-cancro": 157,
  "14.1-20200114-unofficial-cancro": 2,
  "14.1-20200115-unofficial-cancro": 1,
  "14.1-20200116-unofficial-cancro": 2,
  "14.1-20200117-unofficial-cancro": 19,
  "14.1-20200119-unofficial-cancro": 1,
  "14.1-20200120-unofficial-cancro": 62,
  "14.1-20200126-unofficial-cancro": 6,
  "14.1-20200203-unofficial-cancro": 6,
  "14.1-20200205-unofficial-cancro": 158,
  "14.1-20200206-unofficial-cancro": 3,
  "14.1-20200212-unofficial-cancro": 1,
  "14.1-20200217-unofficial-cancro": 1,
  "14.1-20200218-unofficial-cancro": 87,
  "14.1-20200222-unofficial-cancro": 1,
  "14.1-20200226-unofficial-cancro": 1,
  "14.1-20200228-unofficial-cancro": 12,
  "14.1-20200302-unofficial-cancro": 1,
  "14.1-20200303-unofficial-cancro": 4,
  "14.1-20200304-unofficial-cancro": 22,
  "14.1-20200306-unofficial-cancro": 83,
  "14.1-20200310-unofficial-cancro": 2,
  "14.1-20200311-unofficial-cancro": 8,
  "14.1-20200313-unofficial-cancro": 6,
  "14.1-20200314-unofficial-cancro": 4,
  "14.1-20200316-unofficial-cancro": 17,
  "14.1-20200317-unofficial-cancro": 31,
  "14.1-20200318-unofficial-cancro": 4,
  "14.1-20200321-unofficial-cancro": 2,
  "14.1-20200323-unofficial-cancro": 1,
  "14.1-20200326-unofficial-cancro": 1,
  "15.1-20180331-unofficial-cancro": 533,
  "15.1-20180418-unofficial-cancro": 114,
  "15.1-20180809-unofficial-cancro": 1,
  "15.1-20180907-unofficial-cancro": 11,
  "15.1-20180916-unofficial-cancro": 83,
  "15.1-20181020-unofficial-cancro": 8,
  "15.1-20190131-unofficial-cancro": 1,
  "16.0-20181027-unofficial-cancro": 25,
  "16.0-20181101-unofficial-cancro": 134,
  "16.0-20181103-unofficial-cancro": 24,
  "16.0-20190512-unofficial-cancro": 1,
  "16.0-20190717-unofficial-cancro": 22,
  "16.0-20191214-unofficial-cancro": 1,
  "16.0-20191217-unofficial-cancro": 1,
  "16.0-20200112-unofficial-cancro": 1,
  "16.0-20200131-nightly-natrium": 5,
  "17.0-20190918-unofficial-cancro": 11,
  "17.0-20191011-unofficial-cancro": 21,
  "17.0-20191014-unofficial-cancro": 4,
  "17.0-20191016-unofficial-cancro": 1,
  "17.0-20191018-unofficial-cancro": 4,
  "17.0-20191019-unofficial-cancro": 2,
  "17.0-20191020-unofficial-cancro": 24,
  "17.0-20191031-unofficial-cancro": 58,
  "17.0-20191224-unofficial-cancro": 1,
  "17.1-20200125-unofficial-cancro": 1,
  "17.1-20200127-unofficial-cancro": 1,
  "cancro-user 7.1.2mmb29m 3784e03995 release-keys": 1,
  "cancro-user 7.1.2mmb29m 4e4pr42d3j release-keys": 1,
  "cancro-user 7.1.2mmb29m h7976s5i56 release-keys": 1,
  "cancro-user 7.1.2mmb29m hcz8n930iy release-keys": 1,
  "cancro-user 7.1.2mmb29m oi6539h2cv release-keys": 1,
  "cancro-user 7.1.2mmb29m tc3lbuk567 release-keys": 1,
  "cancro-user 7.1.2mmb29m w9h23g1y02 release-keys": 1,
  "cm_cancro-userdebug 5.1 lmy47o aef9ff8fef test-keys": 2,
  "ktu84p": 1,
  "lineage-cancro": 25,
  "lineage_cancro-userdebug 7.1.2 njh47f 00706611ea release-keys": 5,
  "lineage_cancro-userdebug 7.1.2 njh47f 103330bb0e release-keys": 1,
  "lineage_cancro-userdebug 7.1.2 njh47f 155b5c5655 release-keys": 8,
  "lineage_cancro-userdebug 7.1.2 njh47f 1941830316 release-keys": 2,
  "lineage_cancro-userdebug 7.1.2 njh47f 1ea922f073 release-keys": 1,
  "lineage_cancro-userdebug 7.1.2 njh47f 2087af1c7e test-keys": 350,
  "lineage_cancro-userdebug 7.1.2 njh47f 2101d89a84 release-keys": 4,
  "lineage_cancro-userdebug 7.1.2 njh47f 2805c24e76 release-keys": 43,
  "lineage_cancro-userdebug 7.1.2 njh47f 30740210da release-keys": 1,
  "lineage_cancro-userdebug 7.1.2 njh47f 3cf57bed69 release-keys": 1,
  "lineage_cancro-userdebug 7.1.2 njh47f 3dc55b6510 release-keys": 14,
  "lineage_cancro-userdebug 7.1.2 njh47f 3f7b8a5734 release-keys": 1,
  "lineage_cancro-userdebug 7.1.2 njh47f 48b9792c52 release-keys": 50,
  "lineage_cancro-userdebug 7.1.2 njh47f 4b5111ae33 release-keys": 167,
  "lineage_cancro-userdebug 7.1.2 njh47f 5569899641 release-keys": 5,
  "lineage_cancro-userdebug 7.1.2 njh47f 6132113a0f release-keys": 8,
  "lineage_cancro-userdebug 7.1.2 njh47f 738b8258f4 release-keys": 64,
  "lineage_cancro-userdebug 7.1.2 njh47f 782da23e4a release-keys": 75,
  "lineage_cancro-userdebug 7.1.2 njh47f 785173efff release-keys": 1,
  "lineage_cancro-userdebug 7.1.2 njh47f 861c1a59c4 release-keys": 5,
  "lineage_cancro-userdebug 7.1.2 njh47f 94bb4cc8ad release-keys": 128,
  "lineage_cancro-userdebug 7.1.2 njh47f 99bc9ada01 release-keys": 64,
  "lineage_cancro-userdebug 7.1.2 njh47f 9c466573d6 release-keys": 35,
  "lineage_cancro-userdebug 7.1.2 njh47f a37a7035aa release-keys": 1,
  "lineage_cancro-userdebug 7.1.2 njh47f a394dd4b41 release-keys": 51,
  "lineage_cancro-userdebug 7.1.2 njh47f a4401a6b7a release-keys": 450,
  "lineage_cancro-userdebug 7.1.2 njh47f a4a616f93a release-keys": 713,
  "lineage_cancro-userdebug 7.1.2 njh47f a5523b745f release-keys": 3,
  "lineage_cancro-userdebug 7.1.2 njh47f abc4f0a0bc release-keys": 17,
  "lineage_cancro-userdebug 7.1.2 njh47f b2d3a5839e release-keys": 1,
  "lineage_cancro-userdebug 7.1.2 njh47f c902ab8bec release-keys": 1,
  "lineage_cancro-userdebug 7.1.2 njh47f d25909ae99 release-keys": 127,
  "lineage_cancro-userdebug 7.1.2 njh47f d80c90674d release-keys": 14,
  "lineage_cancro-userdebug 7.1.2 njh47f de55e6377a release-keys": 525,
  "lineage_cancro-userdebug 7.1.2 njh47f e9f379b7ee release-keys": 1,
  "lineage_cancro-userdebug 7.1.2 njh47f eb80cf5e4c release-keys": 9,
  "lineage_cancro-userdebug 7.1.2 njh47f ecb466db22 release-keys": 2,
  "lineage_cancro-userdebug 7.1.2 njh47f f50a92041f release-keys": 57,
  "lineage_cancro-userdebug 7.1.2 njh47f f512bc46d3 release-keys": 10,
  "lineage_cancro-userdebug 7.1.2 njh47f f599b1cfdc release-keys": 4,
  "lineageos-cancro": 257,
  "mos-10.0.0-beta1.0-cancro": 1,
  "mos-8.1.0-stable2.0-cancro": 2,
  "mos-9.0.0-beta4.0-cancro": 2,
  "mos-9.0.0-rc3.0-cancro": 1,
  "mos-9.0.0-rc4.0-cancro": 2,
  "mos-9.0.0-stable2.0-cancro": 2,
  "mr_rom oero v0.2 alpha 2018-03-08": 2,
  "msm8953_64-user 7.1.2huaweicaz-al10 b998ntddyj release-keys": 1,
  "os-extended-16.0-20190728-unofficial-cancro": 87,
  "r9sk-user 7.1.2mmb29m 2o1c3f8e81 release-keys": 1,
  "sm_cancro-userdebug 8.1.0 opm1.171019.018 f89c61b351 test-keys": 3,
  "to_cancro-userdebug 7.1.2 njh47f 4222c55623 test-keys": 1,
  "to_cancro-userdebug 7.1.2 njh47f 6aea1d4815 test-keys": 4,
  "xenonhd_cancro-userdebug 8.1.0 opm2.171026.006.h1 54eb6443d9 test-keys": 9,
  "xenonhd_cancro-userdebug 8.1.0 opm2.171026.006.h1 80d5321a1e test-keys": 1
}