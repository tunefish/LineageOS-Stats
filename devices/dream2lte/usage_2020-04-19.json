{
  "14.1-20171005-unofficial-dream2lte": 11,
  "15.1-20190213-unofficial-dream2lte": 3,
  "15.1-20190304-unofficial-dream2lte": 1,
  "16.0-20190325-unofficial-dream2lte": 139,
  "16.0-20190406-unofficial-dream2lte": 7,
  "16.0-20190527-unofficial-dream2lte": 7,
  "16.0-20190618-unofficial-dream2lte": 33,
  "16.0-20190829-unofficial-dream2lte": 43,
  "16.0-20190830-unofficial-dream2lte": 36,
  "16.0-20190922-unofficial-dream2lte": 57,
  "16.0-20191102-unofficial-dream2lte": 459,
  "16.0-20191222-unofficial-dream2lte": 2,
  "16.0-20200104-unofficial-dream2lte": 2,
  "16.0-20200106-unofficial-dream2lte": 12,
  "16.0-20200304-unofficial-dream2lte": 5,
  "16.0-20200305-unofficial-dream2lte": 4,
  "17.0-20191222-unofficial-dream2lte": 210,
  "17.1-20200104-unofficial-dream2lte": 129,
  "17.1-20200108-unofficial-dream2lte": 128,
  "17.1-20200112-unofficial-dream2lte": 327,
  "17.1-20200128-unofficial-dream2lte": 139,
  "17.1-20200131-unofficial-dream2lte": 310,
  "17.1-20200209-unofficial-dream2lte": 44,
  "17.1-20200211-unofficial-dream2lte": 793,
  "17.1-20200212-unofficial-dream2lte": 1
}