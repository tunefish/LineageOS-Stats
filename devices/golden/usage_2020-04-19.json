{
  "13.0-20170118-unofficial-golden": 3,
  "13.0-20170121-unofficial-golden": 7,
  "13.0-20170129-unofficial-golden": 78,
  "13.0-20170224-unofficial-golden": 5,
  "13.0-20170302-unofficial-golden": 13,
  "13.0-20170310-unofficial-golden": 37,
  "13.0-20170328-unofficial-golden": 640,
  "13.0-20170420-unofficial-golden": 31,
  "13.0-20190315-unofficial-golden": 1,
  "14.1-20170122-unofficial-golden": 210,
  "14.1-20170318-unofficial-golden": 5,
  "14.1-20170402-unofficial-golden": 3,
  "14.1-20170415-unofficial-golden": 3,
  "14.1-20170419-unofficial-golden": 6,
  "14.1-20170503-unofficial-golden": 2,
  "14.1-20170507-unofficial-golden": 15,
  "14.1-20170516-unofficial-golden": 1,
  "14.1-20170522-unofficial-golden": 4,
  "14.1-20170609-unofficial-golden": 527,
  "14.1-20170610-unofficial-golden": 9,
  "14.1-20170628-unofficial-golden": 830,
  "14.1-20170724-unofficial-golden": 27,
  "14.1-20170817-unofficial-golden": 65,
  "14.1-20170920-unofficial-golden": 13,
  "14.1-20171011-unofficial-golden": 87,
  "14.1-20171215_192311-official-golden": 76,
  "14.1-20171215_192311-unofficial-golden": 983,
  "14.1-20180109_002536-unofficial-golden": 299,
  "14.1-20181014-unofficial-golden": 1,
  "14.1-20191011-official-golden": 1,
  "14.1-20200222-unofficial-golden": 1,
  "15.1-20181030-unofficial-golden": 7
}